### PR TITLE
Schema Browser Path and minor UI updates

### PR DESCRIPTION
**Pull request by dakota002**

### CODE REVIEW

**Comment by lpsinger** [comment]
Remember that after switching to Remix routing convention v2, we wanted to make use of [nested URLs without layout nesting](https://remix.run/docs/en/main/file-conventions/route-files-v2#nested-urls-without-layout-nesting) so that the schema browser can have a differently layout from the docs themselves. Would you please make that change first?

**Comment by dakota002** [comment]
Ah I see, yea I'll do that

**Comment by dakota002** [comment]
I wrapped the back button in a div to give it some better spacing. 
image
image

Also fixes the size of the version selection button, it is now appropriately in line with the rest of the column

File: app/routes/docs.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Link, NavLink, Outlet } from '@remix-run/react'
 import { GridContainer } from '@trussworks/react-uswds'
 
 import { SideNav, SideNavSub } from '~/components/SideNav'
-import { useActiveLink } from '~/lib/remix'
+// import { useActiveLink } from '~/lib/remix'
```
Comment (lpsinger) [inline app/routes/docs.tsx:12]: Remove commented out code.

```suggestion
```

File: app/routes/docs.tsx [hunk 2]
```diff
@@ -18,137 +18,134 @@ export const handle = {
 
 export default function () {
   const enableSchemaBrowser = useFeature('SCHEMA')
-  const isSchemaBrowser = useActiveLink({ to: '/docs/schema-browser' })
+  // const isSchemaBrowser = useActiveLink({ to: '/docs/schema-browser' })
```
Comment (lpsinger) [inline app/routes/docs.tsx:21]: Remove commented out code.

```suggestion
```

File: app/routes/docs_.schema-browser.tsx [hunk 3]
```diff
@@ -39,41 +44,45 @@ export default function Schema() {
   const { versions } = useLoaderData()
 
   return (
-    <>
-      <Link to="/docs" className="margin-bottom-1">
-        <div className="position-relative">
-          <Icon.ArrowBack className="position-absolute top-0 left-0" />
+    <GridContainer>
```
Comment (lpsinger) [inline app/routes/docs_.schema-browser.tsx:47]:
```suggestion
    <GridContainer className="usa-section">
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — CHANGES_REQUESTED** [review]
Please rebase.

**Comment by dakota002** [comment]
@lpsinger I think it should be good now, the rebase was a bit messy

File: app/routes/docs.tsx (hunk 1 above)
Comment (dakota002) [inline app/routes/docs.tsx:12]:
```suggestion
```

File: app/routes/docs_.schema.$version.$/route.tsx [hunk 4]
```diff
@@ -52,7 +52,6 @@ export async function loader({
 
 export default function () {
   const { version, '*': path } = useParams()
-  // const [showVersions, setShowVersions] = useState(false)
```
Comment (lpsinger) [inline app/routes/docs_.schema.$version.$/route.tsx:55]: There's still a line of commented out code here.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/docs_.schema.$version.$/route.tsx (hunk 4 above)
Comment (dakota002) [reply app/routes/docs_.schema.$version.$/route.tsx:55]: That line should be gone now, just pushed a new commit

**Review by lpsinger — APPROVED** [review]

**Pull request merged**